modules/README.markdown

```diff
@@ -134,4 +134,17 @@ closure arguments.
 	or multiple `excludes` declarations), write them on separate lines.
 	* Order assignments alphabetically, leaving an empty line after multiple
 	calls to the same method (e.g., after multiple `dependsOn` declarations) or if
-	the assignment has a closure.
\ No newline at end of file
+	the assignment has a closure.
+
+## Subrepositories
+
+### `gradle.properties`
+
```

Comment:
That would be perfect! Maybe the link can point directly to https://github.com/liferay/com-liferay-journal/blob/master/gradle.properties.